src/services/shims.ts

```diff
@@ -410,7 +412,7 @@ namespace ts {
             }
         }
 
-        public readDirectory(rootDir: string, extension: string, exclude: string[]): string[] {
+        public readDirectory(rootDir: string, extension: string, exclude: string[], depth?: number): string[] {

```

Comment:
Oops, the implementation was override during merging with master. Will fix soon.